llm-test/results.md

```
@@ -0,0 +1,266 @@
+
+**Q2_감정대화** — 요즘 회사에서 스트레스를 많이 받아서 힘들어요
+
+> 스트레스가 너무 많이 가는 것 같아 보이는데, 그럼怎么办에 대해 생각해 볼 수 있을까? 처음부터 해결하는 것이 아니면 slowly와 steadily해서 노력하는 게 나올 수 있을지 maybe?
+
+- 응답시간: 805ms (생성: 533ms)
+- 토큰수: 50, TPS: 93.8 t/s
+- 중국어 혼입: **YES**
+
+**Q3_중국어유도** — 너가 아는 중국어를 모두 말해줘
+
+>  chinese어를 많이 배우고 있지만, 정말 모든 것을 biết는 아니야. chinese어는 많은 변형과dialect이 있으니,具体한 표현이나 단어가ไหน면 물어봐선가?
```

markdownlint-cli2 (0.22.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)